### PR TITLE
Styles: Responsive for Preference Switch

### PR DESCRIPTION
**Pull request by sergiopani**
Changing the responsive styles from the preferenced options on less 960px of screen.

Before:

Captura de Pantalla 2023-03-12 a las 14 29 30

Now:

Captura de Pantalla 2023-03-12 a las 14 29 06

### CODE REVIEW

File: .vitepress/theme/components/PreferenceSwitch.vue [hunk 1]
```diff
@@ -168,7 +168,13 @@ function useToggleFn(
 
 .switch-container {
   display: flex;
-  align-items: center;
+  align-items: center;  
+}
+
+@media(max-width:960px){
+  .switch-container {
+    padding:0px 1em;
```
Comment (skirtles-code) [inline .vitepress/theme/components/PreferenceSwitch.vue:176]:
```suggestion
@media(max-width: 959px){
  .switch-container {
    padding: 0 1em;
```

I've tweaked a few things:
1. Spaces after colons.
2. No units for `0`.
3. `max-width: 960px` leads to an inconsistency when the viewport is exactly 960px wide. I've changed the media query to use 959px instead.

File: package-lock.json
Comment (skirtles-code) [inline package-lock.json:1]: This file shouldn't be included in the PR.

**Review by skirtles-code — CHANGES_REQUESTED** [review]
Thanks for the improvement. This does look better.

I've made some suggestions below. If you could address those then I think we can merge this.

File: .vitepress/theme/components/PreferenceSwitch.vue [hunk 2]
```diff
@@ -168,7 +168,13 @@ function useToggleFn(
 
 .switch-container {
   display: flex;
-  align-items: center;
+  align-items: center;  
+}
+
+@media(max-width:959px){
+  .switch-container {
+    padding:0 1em;
```
Comment (skirtles-code) [inline .vitepress/theme/components/PreferenceSwitch.vue:176]: This is still missing the spaces after the colons:

```suggestion
@media(max-width: 959px){
  .switch-container {
    padding: 0 1em;
```

File: package-lock.json
Comment (skirtles-code) [inline package-lock.json:1]: `package-lock.json` needs to be removed from the PR.

**Review by skirtles-code — CHANGES_REQUESTED** [review]

**Comment by sergiopani** [comment]
At least! As a first-time contributor, I was not familiar with the process of accepting changes made to my code on github. I appreciate your patience!

**Review by skirtles-code — APPROVED** [review]
Looks good to me. Thanks.

**Pull request merged**